src/main/java/Duke.java

```diff
@@ -22,7 +23,8 @@ public static void main(String[] args) {
                 System.out.println("____________________________________________________________");
```

Comment:
Suggestion:
Put these underscore text into a variable string so your code can be shorter.

Reply:
I agree with this suggestion. You could have a static final variable in duke class as the underscore lines to make the code easier to read